.github/scripts/monday/removeLabel.js

```diff
@@ -22,22 +24,18 @@ module.exports = async ({ context }) => {
   }
 
   const tokensLabels = [designTokens, tokensPackage];
-  const remainingTokenLabel = tokensLabels.includes(labelName) && tokensLabels.find((l) => l !== labelName);
+  const remainingTokenLabel =
```

Comment:
I think you could simplify this to `const remainingTokenLabel = tokensLabels.find((l) => l !== labelName);` since it's checked for truthiness right after.

Reply:
That seems easier to read and understand. Nice!